home2.1/task3.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
```

Comment:
I hope that task3.cpp can be removed from home2.1 folder as it is already a part of class2.1